supporting_content/dimensions/pricingmeasure.md

```diff
@@ -0,0 +1,62 @@
+# Column: PricingMeasure
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                                        | Column                   |
+|:----------|:------------------------------------------------|:-------------------------|
+| AWS       | CUR                                             | pricing/unit             |
+| GCP       | BigQuery Billing Export                         | usage.pricing_unit       |
+| Microsoft | Cost details                                    | UnitOfMeasure            |
+| OCI       | Cost and Usage Report;<br>List Pricing REST API | cost/billingUnitReadable is closest match<br>*Note: The values in this column are similar but not identical to those in the List Pricing REST API; therefore, it may be preferable to rely on the List Pricing REST API, which we already plan to use for ListUnitPrice.* |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider  | Scenario                | Pattern                                                                                                                              |
+|-----------|-------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| AWS       | CUR                     | hours; Hrs; Queries; GB; Secrets; vCPU-Hours; API Requests; Keys; Alarms etc.                                                        |
+| GCP       | BigQuery Billing Export | tebibyte; count; gibibyte; tebibyte; hour; gibibyte hour; gibibyte month; gibibyte hour; gibibyte; hour; count; gibibyte month; etc. |
+| Microsoft | Cost details            | 1 Hour; 10 Hours; 1/Month; 1 GB; 10K; 1 GB-Month; etc.                                                                               |
+| OCI       | Cost and Usage Report   | ONE GiB HOURS STORAGE_SIZE; ONE GiB HOURS MEMORY; etc.                                                                               |
+| OCI       | List Pricing REST API   | Gigabyte Per Hour; etc.<br>*Note: Gigabyte Per Hour corresponds to both ONE GiB HOURS STORAGE_SIZE and ONE GiB HOURS MEMORY*         |
```

Comment:
Both Cost and Usage Report and List Pricing REST API are listed on line 12... Is only the CUR here on line 24 on purpose...?

Reply:
The idea was to highlight that in the case of **OCI**, the values in **billing data** and the **Pricing API** are **aligned but not identical**. (The values in the Pricing API (an alternative OCI data source) represent the exact match.) 

We can use `cost/billingUnitReadable` as a **close match**, but considering that we already need the same API for ListUnitPrice values, we can also retrieve the **exact match** values for PricingMeasure from the **Pricing API**.

Reply:
Okay!  Given the verbiage in line 12, I would expect to see an example of said "close match" in line 24, but it's not a showstopper.  Feel free to resolve if it works for you as-is.

Reply:
- In line 23 there are two 'close match' values (values from the OCI cost and usage report) 
- In line 24 there is the exact match, i.e. what we would expect to get (value from the Pricing API).

Reply:
Geez.  I'm sorry Irena... I don't know how I missed that.  Sorry to waste your time... amateur hour over here.